### PR TITLE
feat: add python flatbuffers pip package files.

### PR DESCRIPTION
**Pull request by shuaijinchao**
@spacewander please review.

### CODE REVIEW

**Comment by tzssangglass** [comment]
`python/setup.py` is a manually written document? AFAIK, this should only hold files generated by `flatc` compilation. But I don't know python well enough to be sure if this is appropriate.

**Comment by shuaijinchao** [comment]
> `python/setup.py` is a manually written document? AFAIK, this should only hold files generated by `flatc` compilation. But I don't know python well enough to be sure if this is appropriate.

Yes, `setup.py` is a package file that uploads the code to the official pip warehouse. The purpose of putting it in the `python` directory is to not pollute other directory environments when performing packaging.

File: python/setup.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+from setuptools import setup
+from setuptools import find_packages
+
+setup(
+    name="a6pluginproto",
+    version="0.2.0",
```
Comment (spacewander) [inline python/setup.py]: Why start from 0.2?

File: python/setup.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    description="Proto of APISIX Python plugin runner.",
+    license="Apache License 2.0",
+    url="https://github.com/api7/ext-plugin-proto",
+    author="Jinchao Shuai",
+    author_email="shuaijinchao@apache.org",
+    packages=find_packages(),
+    install_requires=["flatbuffers", "minicache", "click"]
```
Comment (spacewander) [inline python/setup.py]: why depends on these libraries?

File: python/setup.py (hunk 1 above)
Comment (spacewander) [reply python/setup.py]: We have enough change to release the 0.2 after merging the https://github.com/api7/ext-plugin-proto/pull/14

**Comment by shuaijinchao** [comment]
@spacewander the version and dependencies have been corrected, please review again.

**Comment by tzssangglass** [comment]
> > `python/setup.py` is a manually written document? AFAIK, this should only hold files generated by `flatc` compilation. But I don't know python well enough to be sure if this is appropriate.
> 
> Yes, `setup.py` is a package file that uploads the code to the official pip warehouse. The purpose of putting it in the `python` directory is to not pollute other directory environments when performing packaging.

**Comment by tzssangglass** [comment]
> > `python/setup.py` is a manually written document? AFAIK, this should only hold files generated by `flatc` compilation. But I don't know python well enough to be sure if this is appropriate.
> 
> Yes, `setup.py` is a package file that uploads the code to the official pip warehouse. The purpose of putting it in the `python` directory is to not pollute other directory environments when performing packaging.

I got.

**Review by spacewander — APPROVED** [review]

**Pull request merged**